# for resetting tracking id if video loop detected

std_msgs/Header header
bool reset_triggered 



# Ref:https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Single-Package-Define-And-Use-Interface.html# 
